code/src/components/Today.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components'
+
+const Today = () => {
+  let today = new Date()
+  let dd = today.getDate()
+  let mm = today.getMonth() + 1
+  let yyyy = today.getFullYear()
+  if (dd < 10) { dd = 0+dd}
+  if (mm < 10) {mm = 0+mm}
+  today = dd +'/'+dd+'/'+yyyy
```

Comment:
Nice that you managed to get the date in the format you wished by specifying like this! Maybe it would be more convenient though to import some library like moment here to be able to easier format the date? I also wonder about the use of let instead of const. Why is it necessary here? 🤔

Reply:
Yessss, [moment.js](https://momentjs.com/) is awesome for that!
Otherwise, I think this snippet of code needs some explanation. 10/10/2020 🤔

Reply:
These codes are to practice printing out dates without using available library like moment.js. I use let rather than const because the value will be reasigned.